tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -11624,6 +11633,15 @@ declare namespace ts.codefix {
 }
 declare namespace ts.codefix {
 }
+declare namespace ts.codefix {
+    class Deleter {
```

Comment:
where is this defined? i can not see to find it in this PR?

Reply:
Bottom of `fixUnusedIdentifier.ts`. It's internal but those are currently being included in the API (#24966).